.babelrc-cjs.js

```diff
@@ -0,0 +1,9 @@
+const baseConfig = require('./.babelrc');
+
+const config = { ...baseConfig };
```

Comment:
Perhaps a helpful comment on the module level here to explain what it's doing (overriding the `modules` setting...etc.).